crypto/cversion.c

```diff
@@ -43,6 +44,30 @@ const char *OPENSSL_version_build_metadata(void)
     return OPENSSL_VERSION_BUILD_METADATA_STR;
 }
 
+static char ossl_cpu_info_str[80];
+static CRYPTO_ONCE init = CRYPTO_ONCE_STATIC_INIT;
+
+DEFINE_RUN_ONCE_STATIC(ossl_init_cpu_info_str)
+{
+#if     defined(__i386)   || defined(__i386__)   || defined(_M_IX86) || \
+        defined(__x86_64) || defined(__x86_64__) || \
+        defined(_M_AMD64) || defined(_M_X64)
+    sprintf(ossl_cpu_info_str,
+# ifdef _WIN32
```

Comment:
one clang travis build spec has a warning here:
`embedding a directive within macro arguments has undefined behavior [-Werror,-Wembedded-directive]`
It does not make much sense to me, is sprintf a macro?
I will try to fix by moving the directive before the sprintf

Reply:
You might look at what bn_lcl.h does for printing 64bit longs

Reply:
Hmm, maybe I try BIO_snprintf